anchore_engine/services/policy_engine/engine/policy/gates/vulnerabilities.py

```diff
@@ -282,6 +296,11 @@ def evaluate(self, image_obj, context):
                                 parameter_data["pkg_class"] = "non-os"
                                 parameter_data["pkg_type"] = image_cpe.pkg_type
 
+                                if package_path_re is not None:
+                                    match_found = package_path_re.match(image_cpe.pkg_path)
+                                    if match_found is not None:
+                                        continue
```

Comment:
I would consider adding at least a debug log here. When a package gets excluded by a path it might be helpful for troubleshooting later on.